src/material/dialog/dialog-content-directives.ts

```diff
@@ -145,9 +145,15 @@ export class MatDialogContent {}
  */
 @Directive({
   selector: `[mat-dialog-actions], mat-dialog-actions, [matDialogActions]`,
-  host: {'class': 'mat-dialog-actions'},
+  host: {
+    'class': 'mat-dialog-actions',
+    '[class.mat-dialog-actions-align-center]': 'align === "center"',
+    '[class.mat-dialog-actions-align-end]': 'align === "end"',
+  },
 })
-export class MatDialogActions {}
+export class MatDialogActions {
+  @Input() align?: 'center' | 'end' = undefined;
```

Comment:
Rather than `undefined`, should it support `start` which doesn't apply any styling?

Reply:
Ok, I will resubmit a new PR.

Yes, that might be better for API consistency. I will do that change in my next PR